build.gradle

```diff
@@ -46,31 +46,35 @@ project.ext {
     minSdkVersion = 16
     targetSdkVersion = 28
 
+    // AndroidX
+    annotationVersion = "1.1.0"
+    appcompatVersion = "1.1.0"
+    mediaVersion = "1.1.0"
+    preferenceVersion = "1.1.1"
     workManagerVersion = "2.3.4"
-    espressoVersion = "3.2.0"
-    awaitilityVersion = "3.1.6"
-    commonsioVersion = "2.5"
+    googleMaterialVersion = "1.1.0"
+
+    // Third-party
     commonslangVersion = "3.6"
-    eventbusVersion = "3.2.0"
-    glideVersion = "4.8.0"
-    glideOkhttpIntegrationVersion = "4.8.0"
-    iconifyVersion = "2.2.2"
+    commonsioVersion = "2.5"
     jsoupVersion = "1.11.2"
+    glideVersion = "4.8.0"
     okhttpVersion = "3.12.10"
     okioVersion = "1.17.5"
-    recyclerviewFlexibledividerVersion = "1.4.0"
-    robotiumSoloVersion = "5.6.3"
+    eventbusVersion = "3.2.0"
```

Comment:
The definitions were (more or less) sorted alphabetically. Why did you change the order of the third-party ones?

Reply:
I was matching it with the order of the third-party dependencies in the module gradles. Also grouping them, like testing together and such.